tasks/generate.py

```diff
@@ -29,12 +26,12 @@ def announce(ctx, version):
     contributors_text = '\n'.join('* {}'.format(name) for name in sorted(contributors)) + '\n'
     text = template_text.format(version=version, contributors=contributors_text)
 
-    target = Path(__file__).joinpath('../../doc/en/announce/release-{}.rst'.format(version))
+    target = Path(__file__).parent.joinpath('../doc/en/announce/release-{}.rst'.format(version))
```

Comment:
Oh just realized, do you mean why not `.parent.parent.joinpath('doc/` instead of `.parent.joinpath('../doc`? Mostly I find the latter shorter.